samples/meetings-content-bubble/csharp/assets/sample.json

```diff
@@ -3,10 +3,10 @@
     "name": "officedev-microsoft-teams-samples-meetings-content-bubble-csharp",
     "source": "officeDev",
     "title": "Content Bubble Meeting app",
-    "shortDescription": "Illustrates how to implement the \"content bubble\" in-meeting experience",
+    "shortDescription": "Illustrates how to implement the \"content bubble\" and \"targeted notifications\" in-meeting experience",
```

Comment:
I think the description is slightly confusing :) We are rebranding the existing "content bubble" to be "in-meeting notification". This feature is to send notification to everyone in the meeting. 

For the new feature, we want to send the in-meeting notification only to targeted participants, and this bot is demonstrating this feature :) We call this feature as 'targeted in-meeting notification'. 

With these feature names, can we update the description?

Reply:
Sure, updated the description

Reply:
Hey @Prithvi-MSFT , I still see "content bubble" here :) 
This is the description that I am thinking. 
"Illustrates how to implement the \"in-meeting notification\" and \"targeted meeting notification\" experience".

In-meeting notification (previously called, content bubble): https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#send-an-in-meeting-notification

Targeted Meeting notification: https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api